### ISSUE TITLE
[DESIGN]: GIBCT EYB - CONSIDER re-aligning the benefits list items for VETTEC providers

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#14422]
# [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a user, I want to skim the benefit amounts quickly. The VETTEC estimated benefits columns feel like they could be re-aligned to improve this readability.

## Details

The VETTEC estimated benefits amounts are floating to the left, causing the right-most column to be less readable than if the numbers were all aligned right. Screenshot attached below.

/cc @CrystabelReiter && @emilywaggoner for VSP design awareness 

## Acceptance Criteria

- [ ] Right column is aligned consistently on desktop and mobile

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Environment

* https://staging.va.gov/gi-bill-comparison-tool/profile/3V000142/CERTIFIED%20INFORMATION%20SYSTEMS%20SECURITY or other VETTEC profile view

## Screenshots or Trace Logs

![Screen Shot 2020-10-05 at 4 05 53 PM](https://user-images.githubusercontent.com/934879/95132463-6daa6500-0725-11eb-8c80-c2eeb9798674.png)

**Comment by desireeturner** [comment]
@JenLJ @watersemma @amyknox looking for your input on this issue. Thank you!

**Comment by amyknox** [comment]
Hi @1Copenut - this is an oldie! I get your intent that the numbers should be lined up but based on the calculation, the items which are indented from the right margin are sub-calculations on for when the provider gets its payments. I believe adding them to the right aligned column would make that calculation more challenging for users to scan.  However, given your experience, it must be a little tricky for users to understand the intent of the layout. 

We'll take a closer look at the presentation of these calculations during the on-going redesign effort. 
![image](https://user-images.githubusercontent.com/41081945/101407317-b22eba00-38a8-11eb-97d3-dcb328c9d625.png)
